core-feature-pack/common/src/main/resources/modules/system/layers/base/org/wildfly/extension/core-management/main/module.xml

```diff
@@ -30,7 +30,7 @@
         <module name="org.wildfly.extension.core-management-client"/>
         <module name="org.wildfly.security.elytron-private"/>
         <module name="org.wildfly.service"/>
-        <module name="org.wildfly.unstable.annotation.api.indexer"/>
-        <module name="org.wildfly._internal.unstable-api-annotation-index"/>
+        <module name="org.wildfly.unstable.annotation.api.indexer" optional="true"/>
```

Comment:
I'm not totally familiar with how this works.
Does being optional AND the dependency having `jboss.stability=preview` mean it gets provisioned?

Reply:
When core-management is transformed onto a Galleon package, dependencies on optional modules are ignored. An optional module is expected to be provisioned from another root. For example, a layer, a subsystem, ...

Reply:
Looking for 'org.wildfly._internal.unstable-api-annotation-index' and 'org.wildfly.unstable.annotation.api.indexer' in the wildfly-core sources, I don't see those being referenced anywhere that could make a difference apart from  via this module dependency

Reply:
I've added the packages via model.xml, let's see how it goes